include/tvm/relay/module.h

```diff
@@ -111,6 +115,27 @@ class ModuleNode : public RelayNode {
    */
   void Update(const Module& other);
 
+  /*!
+   * \brief Get the entry point of the module.
+   *
+   * \returns The entry point function, (i.e. main).
+   */
+  Expr EntryPoint();
```

Comment:
This name could still use more discussions. 

Given that this is an one-liner module[module->entry_func], maybe we don't need this function

Reply:
What about `Main`?

Reply:
remove this